src/rgw/support/rgw_support_bundle

```diff
@@ -60,11 +60,9 @@ class RGWSupportBundle:
             RGWSupportBundle._cleanup()
         os.makedirs(RGWSupportBundle._tmp_src, exist_ok=True)
         # copy configuration files
-        try:
+        if os.path.exists(config_file):
             shutil.copyfile(config_file,\
                 os.path.join(RGWSupportBundle._tmp_src, RGW_CONF_FILE))
```

Comment:
If config file is not present, are we not logging it?

Reply:
Added Logging for rgw support bundle